### PR TITLE
Support struct

### PR DESCRIPTION
**Pull request by pgrivachev**
- Added struct type
- Refactored
- Changed tests framework to mocha

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -4,20 +4,6 @@
...
-        "--inspect-brk",
-        "node_modules/.bin/jest",
-        "--runInBand",
-        "--config",
-        "jestconfig.json"
-      ],
-      "console": "integratedTerminal",
-      "internalConsoleOptions": "neverOpen"
-    },
     {
```
Comment (fivetran-dmitryboykov) [inline .vscode/launch.json:7]: How to debug tests?

File: src/test/SimpleColumn.spec.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { SimpleColumn } from '../SimpleColumn';
+import { SimpleType } from '../SimpleType';
+import { TypeFactory } from '../TypeFactory';
+import { TypeKind } from '../types/zetasql/TypeKind';
+
+describe('SimpleColumnTest', () => {
+  it('testSimpleTypeColumn', () => {
+    const type = TypeFactory.createSimpleType(TypeKind.TYPE_BOOL);
+    if (!type) {
+      throw new Error('Simple type expected');
```
Comment (fivetran-dmitryboykov) [inline src/test/SimpleColumn.spec.ts:11]: `assertThat(type, defined());`  ?

File: .eslintrc.json [hunk 3]
```diff
@@ -2,17 +2,67 @@
   "root": true,
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaVersion": 2019,
-    "sourceType": "module"
+    "ecmaVersion": "es2019",
```
Comment (fivetran-dmitryboykov) [inline .eslintrc.json:5]: `es2021` ?

**Review by fivetran-dmitryboykov — COMMENTED** [review]
Looks good. Please answer the questions.

File: .vscode/launch.json (hunk 1 above)
Comment (pgrivachev) [reply .vscode/launch.json:7]: Use https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-mocha-test-adapter extension. Actually I thought you already use it.

File: .eslintrc.json (hunk 3 above)
Comment (pgrivachev) [reply .eslintrc.json:5]: Good catch!

**Review by fivetran-dmitryboykov — APPROVED** [review]

**Pull request merged**